python_frank_energie/frank_energie.py

```
@@ -19,251 +45,999 @@ class FrankEnergie:
         if self._session is None:
             self._session = ClientSession()
             self._close_session = True
 
         try:
-            resp = await self._session.post(
+            # print(f"Request: POST {self.DATA_URL}")
+            headers = {
+                "Content-Type": "application/json",
+                "Authorization": f"Bearer {self._auth.authToken}"
+            } if self._auth is not None else None
+            sanitized_query = sanitize_query(query)
```

Ruff

Local variable `sanitized_query` is assigned to but never used

Remove assignment to unused variable `sanitized_query`

(F841)